### DISCUSSION TITLE
Using wavy sliders?

### DISCUSSION THREAD

**Discussion by mahozad** [sayaka-sh/spmp#224] [Ideas]
Hi,

I'm doing a self-promotion here :)
Please ignore/close it if you are not interested.

Would you like to use the [wavy-slider](https://github.com/mahozad/wavy-slider) in your app?
I think Android 13 uses this kind of slider for its media controls.

[Demo](https://mahozad.ir/wavy-slider).

**Comment by sayaka-sh** [comment]
Thanks, but I don't really have a need for this at the moment
